device/dhcp/clients/v4/dhcpv4client.yaml

```diff
@@ -0,0 +1,73 @@
+components:
+  schemas:
+    Device.Dhcpv4client:
+      description: >-
+        Configuration for emulated DHCPv4 Client on a single Interface.
+        https://www.rfc-editor.org/rfc/rfc2131.html
+      type: object
+      x-status:
+        status: under_review
+      required: [name]
+      properties:
+        name:
+          x-include: ../../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        selecting_server:
+          description: >-
+            The client receives one or more DHCPOFFER messages from one or more servers. The client may choose to wait for multiple responses.
+            The client chooses one server from which to request configuration
+            parameters, based on the configuration parameters offered in the DHCPOFFER messages.
+            use_first_server: if selected, the subnet accepts the IP addresses offered by the first server to respond with an offer of IP addresses.
+            use_server_address: The address of the DHCP server from which the subnet will accept IP addresses.
+            server address to 0.0.0.0, implementation will use the services of the first DHCP server that responds to the DHCP Lease Request message.
```

Comment:
This is not clear , why not have a separate option if this is different from both the options ?